storage/migrations/mysql/1_init.sql

```diff
@@ -87,158 +86,164 @@ CREATE TABLE IF NOT EXISTS `templates`
 ) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
 
 CREATE TABLE IF NOT EXISTS `campaigns` (
-  `id`            INTEGER UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
-  `user_id`       INTEGER UNSIGNED NOT NULL,
-  `name`          VARCHAR(191) NOT NULL,
-  `template_id`   INTEGER UNSIGNED,
-  `status`        VARCHAR(191) NOT NULL,
-  `created_at`    DATETIME(6) NOT NULL,
-  `updated_at`    DATETIME(6) NOT NULL,
-  `completed_at`  DATETIME(6) DEFAULT NULL,
-  `deleted_at`    DATETIME(6) DEFAULT NULL,
-  `started_at`    DATETIME(6) DEFAULT NULL,
-  FOREIGN KEY (`user_id`) REFERENCES users(`id`),
-  FOREIGN KEY (`template_id`) REFERENCES templates(`id`) ON DELETE SET NULL ON UPDATE CASCADE,
-  INDEX id_created_at (`id`, `created_at`)
+    `id`            INTEGER UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
+    `user_id`       INTEGER UNSIGNED NOT NULL,
+    `name`          VARCHAR(191) NOT NULL,
+    `template_id`   INTEGER UNSIGNED,
+    `status`        VARCHAR(191) NOT NULL,
+    `created_at`    DATETIME(6) NOT NULL,
+    `updated_at`    DATETIME(6) NOT NULL,
+    `completed_at`  DATETIME(6) DEFAULT NULL,
+    `deleted_at`    DATETIME(6) DEFAULT NULL,
+    `started_at`    DATETIME(6) DEFAULT NULL,
+    FOREIGN KEY (`user_id`) REFERENCES users(`id`),
+    FOREIGN KEY (`template_id`) REFERENCES templates(`id`) ON DELETE SET NULL ON UPDATE CASCADE,
+    INDEX idx_id_created_at (`id`, `created_at`),
+    INDEX idx_user_id (`user_id`)
```

Comment:
Since the user_id is a foreign key, an index is automatically created we don't need it here.

Reply:
done